.github/workflows/examples.yml

```diff
@@ -38,6 +38,14 @@ jobs:
     timeout-minutes: 120
     if: github.event_name != 'schedule' || github.repository == 'mlflow/mlflow'
     steps:
+      - name: Free Ubuntu Disk Space
+        uses: jlumbroso/free-disk-space@main
```

Comment:
Can we pin the version?

Reply:
good idea. added explicit version pin since updates are pretty infrequent.